utils/geom.py

```diff
@@ -907,6 +907,27 @@ def interpolate_quadratic_bezier(knot1, handle, knot2, resolution):
     handle2 = handle + (1.0/3.0) * (knot2 - handle)
     return interpolate_bezier(knot1, handle1, handle2, knot2, resolution)
 
+def calc_normal(vertices):
```

Comment:
@portnov 

utils/modules/geom_utils.py  has a "obtain_normal3" function

i'm probably being thick, but.. where is function`center` defined?

Reply:
Interesting question. Will look into this.

Reply:
`center` function was not ported from master.
As for duplicating function... it was duplicating in master. It would be good to remove duplication, but I do not consider it is a big problem.

Reply:
`obtain_normal3` function seem to be working with tris only. So I'd suggest to deprecate it and use `calc_normal` instead.

Reply:
i use `obtain_normal3` for my own stuff, so i'd like to keep it.